lib/Backend/LowerMDShared.cpp

```diff
@@ -8851,7 +8851,10 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
                 sharedBailout = (instr->GetBailOutInfo()->bailOutInstr != instr) ? true : false;
                 if (sharedBailout)
                 {
-                    bailoutLabel = instr->GetBailOutInfo()->bailOutInstr->AsLabelInstr();
+                    // In case of a shared bailout, we should jump to the code that sets some data on the 
```

Comment:
I only see an extra label created here, what is "the code that sets some data on the bailout record"?